addons/mail/static/src/scss/composer.scss

```diff
@@ -61,7 +61,8 @@
     }
 }
 
-.o_mail_composer_form {
+.o_mail_composer_form .o_content > .o_form_editable,
```

Comment:
I don't understand that transormation

Reply:
before: the o_mail_composer_form class would have been added to the o_form_editable (or o_form_readonly) div.
after: we should make this selector match the same html element in order to limitate the possible style breakage